### DISCUSSION TITLE
Spell check suggests nonsense hash

### DISCUSSION THREAD

**Discussion by teachamantofish** [streetsidesoftware/vscode-spell-checker#674] [X - Closed]
I've noticed the following: 

- When I type 'the', the ext suggests 'thermosiphon' and 1vTe8DbZzaTUQ_3v4sf_NrHtSr100pS2MrdIHCFQXsMF0I2lxUgUsN53Sd24asb0ODPBk3
- When I add an x to the word above, I see only the hash. As I add letters, if that letter is in that hash, it remains as the suggested word. 
- If I type just an 'x', the suggestion is sXJCNP8mR
- If I type lklk, I get there nonsense suggestions with 'lklk' in it. 
- The nonsense suggestions disappear if I type a space after my word. 

Perhaps this is not the right tool for spell checking a markdown file?

**Comment by Jason3S** [comment]
I do not believe these suggestions are coming from the spell checker.
